cluster/addons/dns/coredns.yaml.base

```diff
@@ -93,9 +97,21 @@ spec:
           effect: NoSchedule
         - key: "CriticalAddonsOnly"
           operator: "Exists"
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
```

Comment:
There is an on-going discussion (https://github.com/kubernetes/kubernetes/pull/57683) about whether to use `preferredDuringSchedulingIgnoredDuringExecution` or `requiredDuringSchedulingIgnoredDuringExecution`. Could we add the anti-affinity bit after that?

Reply:
From https://github.com/kubernetes/kubernetes/pull/57683 seems like we agreed on going with `preferredDuringSchedulingIgnoredDuringExecution` at this point.